server/services/exhibitions.js

```diff
@@ -28,6 +38,12 @@ export async function getExhibition(id: string, previewReq: ?Request): Promise<?
   const standfirst = bodyParts.find(p => p.type === 'standfirst');
   const imageGallery = bodyParts.find(p => p.type === 'imageGallery');
 
+  const relatedContent = exhibition.data.related;
```

Comment:
As mentioned in the content model, I really think the word related is naming it something we wish it was, but isn't.

Would something like `promoLinks` make more sense?

Reply:
As discussed will make this relatedPromos

Reply:
Will do this in another PR after the demo